renderer/types/actions/editor-actions.ts

```diff
@@ -67,4 +77,8 @@ export interface EditorActions {
   downloadCode: () => DownloadCodeAction;
 
   uploadCode: () => UploadCodeAction;
+
+  updateKeyboardBitmap: (keyboardBitmap: number) => UpdateKeyboardBitmapAction;
+
+  updateIsKeyboardModeToggled: (isKeyboardToggled: boolean) => UpdateIsKeyboardModeToggledAction;
```

Comment:
Noticed we have a `updateIsKeyboardModeToggled` Redux action but it isn't actually called anywhere, think we need to add that